libp2p/Session.cpp

```diff
@@ -22,18 +22,30 @@ Session::Session(Host* _h, unique_ptr<RLPXFrameCoder>&& _io, std::shared_ptr<RLP
     m_socket(_s),
     m_peer(_n),
     m_info(_info),
-    m_ping(chrono::steady_clock::time_point::max()),
-    m_logContext(_info.id.abridged() + "|" + _info.clientVersion)
+    m_ping(chrono::steady_clock::time_point::max())
 {
+    stringstream remoteInfoStream;
+    remoteInfoStream << "(" << m_info.id << "@" << m_socket->remoteEndpoint() << ")";
+
+    m_netLogger.add_attribute(
+        "Suffix", boost::log::attributes::constant<std::string>(remoteInfoStream.str()));
```

Comment:
Maybe `auto const attr = boost::log::attributes::constant<std::string>(remoteInfoStream.str());` could help to have less repetition here

Reply:
Good idea, will fix